src/routes/dashboard/pharm/[slug]/states.svelte.ts

```
@@ -34,6 +34,21 @@ export class QuestionMap {
 	questionButtons: HTMLButtonElement[] = $state([]);
 	noFlags = $state(false);
 	navigationIds = $state<string[]>([]);
+	dirtyQuestions = new Set<string>();
+
+	// Debounce function
+	private debounce = (func: Function, delay: number) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)